oidc.go

```diff
@@ -99,6 +99,11 @@ func HandleOpenIDFlow(clientID, clientSecret, scopes, callbackURL, discoveryEndp
 	query.Set("scope", scopes)
 	query.Set("redirect_uri", callbackURL)
 
+	for _, arg := range customArgs {
+		kv := strings.SplitN(arg, "=", 2)
+		query.Set(kv[0], kv[1])
```

Comment:
We are missing input validation in more places but I guess we might want to parse the CLI inout and save the results in a map which would push the validation up.